extensions/sql-database-projects/package.json

```diff
@@ -426,6 +435,11 @@
           "when": "view == dataworkspace.views.main && viewItem == databaseProject.itemType.folder || viewItem =~ /^databaseProject.itemType.file/ || viewItem == databaseProject.itemType.reference || viewItem == databaseProject.itemType.sqlcmdVariable",
           "group": "9_dbProjectsLast@2"
         },
+        {
+          "command": "sqlDatabaseProjects.rename",
+          "when": "view == dataworkspace.views.main && viewItem =~ /^databaseProject.itemType.file/",
```

Comment:
Not for folders?  Or maybe I've just forgotten how the conditional here works.

Reply:
only files for now. I checked the change I made in DacFx and it only supported moving files, so I'll have to go back and add support for moving folders.